DeAceroDLabs/marketplace-frontend
# [DMP-749] Transform JSON to description

## What is done

[[DMP-749] Transformar JSON a descripción de producto](https://deacero.atlassian.net/browse/DMP-749)

## How is done

1. Added a new component: `ProductDescription`.
2. The component will receive a JSON as a prop.
3. For now it uses a dummy JSON to validate.
4. It converts the JSON keys into the section titles, and its values to a paragraph (if they are a string) or a list (if an array)

## How to test

1. Run the project via `npm start`
5. Click on any product in the catalog.
6. You will see the description always visible (and always the same) at the bottom of the product.
7. Check that each section is divided.
